<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="moveX"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="moveY"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="moveZ"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="moveBtn">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QDoubleSpinBox" name="rotateX"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rotateY"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="rotateZ"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="rotateBtn">
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QDoubleSpinBox" name="scaleX"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="scaleY"/>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="scaleZ"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="scaleBtn">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>moveX</tabstop>
  <tabstop>moveY</tabstop>
  <tabstop>moveZ</tabstop>
  <tabstop>moveBtn</tabstop>
  <tabstop>rotateX</tabstop>
  <tabstop>rotateY</tabstop>
  <tabstop>rotateZ</tabstop>
  <tabstop>rotateBtn</tabstop>
  <tabstop>scaleX</tabstop>
  <tabstop>scaleY</tabstop>
  <tabstop>scaleZ</tabstop>
  <tabstop>scaleBtn</tabstop>
 </tabstops>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
